spec/classes/init_spec.rb

```diff
@@ -168,7 +168,13 @@
           })
         }
 
-        it { should_not contain_host('monkey.example.com') }
+        it {
```

Comment:
if the fqdn entry is false, then the catalog should not include the fqdn host entry. Not sure why this was changed.